marketplace/enrich.schema.json

```diff
@@ -9,24 +9,193 @@
+                    "properties": {
+                        "enabled": {
+                            "type": "boolean",
+                            "default": false
+                        }
+                    },
+                    "required": ["enabled"]
                 },
                 "electricCo2Emission": {
-                    "type": "boolean",
-                    "default": true
+                    "type": "object",
+                    "additionalProperties": false,
+                    "description": "Enabled/disable electric co2 emission enricher",
+                    "properties": {
+                        "enabled": {
+                            "type": "boolean",
+                            "default": true
+                        }
+                    },
+                    "required": ["enabled"]
                 },
                 "pictogram": {
-                    "type": "boolean",
-                    "default": true
+                    "type": "object",
+                    "additionalProperties": false,
+                    "description": "Enabled/disable pictogram enricher",
+                    "properties": {
+                        "enabled": {
+                            "type": "boolean",
+                            "default": true
+                        }
+                    },
+                    "required": ["enabled"]
                 },
                 "priceComputation": {
-                    "type": "boolean",
-                    "default": false
+                    "type": "object",
+                    "additionalProperties": false,
+                    "description": "Enabled/disable price computation enricher",
+                    "properties": {
+                        "enabled": {
+                            "type": "boolean",
+                            "default": true
+                        }
+                    },
+                    "required": ["enabled"]
+                },
+                "pictureOptimizer": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "description": "Enabled/disable picture optimizer enricher",
+                    "properties": {
+                        "enabled": {
+                            "type": "boolean",
+                            "default": false,
+                            "description": "By default, picture optimization is disabled for all partners"
+                        },
+                        "crop": {
+                            "type": "object",
+                            "additionalProperties": false,
+                            "properties": {
+                                "enabled": {
+                                    "type": "boolean",
+                                    "default": false
+                                }
+                            },
+                            "required": [
```

Comment:
cometic : short arrays (length to be defined) should maybe be written on a single row. What do you think ?